bin/ivar_variants_to_vcf.py

```diff
@@ -221,165 +242,159 @@ def ivar_variants_to_vcf(FileIn, FileOut, passOnly=False, minAF=0,NotStrandBias=
                     var_type = "DEL"
 
                 QUAL = "."
-                pass_test = line[13]
-                REF_CODON = line[15]
-                ALT_CODON = line[17]
 
-                if NotStrandBias:
-                    if pass_test =="TRUE":
+                ## Determine FILTER field
+                INFO = f"DP={line[11]}"
+                pass_test = line[13]
+                if ignore_strand_bias:
+                    if pass_test == "TRUE":
                         FILTER = "PASS"
                     else:
                         FILTER = "FAIL"
-                    INFO = "DP=" + line[11]
                 else:
-                    # If strand-bias test significative add SB in the FILTER field, else PASS
-                    if p<0.05 and pass_test =="TRUE":
+                    ## Add SB in the FILTER field if strand-bias p-value is significant
+                    if pvalue < 0.05 and pass_test == "TRUE":
                         FILTER = "SB"
-                    elif p>0.05 and pass_test =="TRUE":
+                    elif pvalue > 0.05 and pass_test == "TRUE":
                         FILTER = "PASS"
-                    elif  p<=0.05 and pass_test == "FALSE":
-                        FILTER = "SB,other"
+                    elif  pvalue <= 0.05 and pass_test == "FALSE":
+                        FILTER = "FAIL"
```

Comment:
I have replaced this with `FAIL` instead of `SB,other`. Any other suggestions as to what we can call it to be more specific as you mentioned on Slack? https://nfcore.slack.com/archives/C0110R22NH3/p1643882604079859?thread_ts=1643882394.894439&cid=C0110R22NH3

Reply:
I think it can't be `FAIL` there, in the `FILTER` field there should be all the filters that have failed, in this case as it is right now it should be `SB,FAIL`
FAIL is set when it not passes the p-value for calling in ivar right? we could called it just `FISHER_TEST` ? It is confusing with the strand-bias fisher-test, because they are the same one checking ref vs alt (pvalue ivar) and one checking forward vs reverse (pvalue strandbias)
What do you think?

Reply:
Added `FAIL,SB` in https://github.com/nf-core/viralrecon/pull/272/commits/94fa8ff2cad0d1aea4556f2dab5471874e79c4b2.